Libplanet/Action/RandomExtension.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Libplanet.Action
+{
+    /// <summary>
+    /// This extension class provides some convenient methods
+    /// to deal with <see cref="IRandom"/>.
+    /// </summary>
+    public static class RandomExtension
+    {
+        /// <summary>
+        /// Generates a random <see cref="Guid"/>.
+        /// </summary>
+        /// <param name="random"> <see cref="IRandom"/> to generate
+        /// a random <see cref="Guid"/>.</param>
+        /// <returns> Generated random <see cref="Guid"/>.
+        /// </returns>
+        /// <seealso cref="IRandom"/>
+        public static Guid GenerateRandomGuid(this IRandom random)
+        {
+            var bytes = new byte[16];
```

Comment:
It would be better if there is mention as [this](https://github.com/planetarium/libplanet/blob/714e241/Libplanet/Action/ActionContext.cs#L38).

Reply:
Both are fixed! Thanks!

Reply:
@Unengine Still not addressed I think?

Reply:
Sorry, I forgot to commit. I'll also commit your suggestions at once!

Reply:
Job's done!